src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -463,8 +460,7 @@ class LeVADocumentUseCase extends DocGenUseCase {
                         systemRequirement : testIssue.requirements.join(", "),
                         success           : testIssue.isSuccess ? "Y" : "N",
                         remarks           : testIssue.isMissing ? "Not executed" : "N/A",
-                        softwareDesignSpec: testIssue.getTechnicalSpecifications().findAll{ it.softwareDesignSpec } ?
-                                            testIssue.getTechnicalSpecifications().findAll{ it.softwareDesignSpec }.collect{ it.key }.join(", ") : "N/A",
+                        softwareDesignSpec: (testIssue.techSpecs.join(", "))?: "N/A",
```

Comment:
I don't think this is correct. Here, we purposedly only included references to those techSpecs which contain a softwareDesisgnSpec. Thoughts?

Reply:
I think this filter made no sense since we should be able to reference any techSpec in a test. As long as they are linked, I see no issue with this.

Reply:
Hmm. But the property is called softwareDesignSpec, not techSpec. Therefore, I would assume it contains a reference to 1..N design specs. The best reference we can give is a techSpec, since the softwareDesignSpec is contained therein. However, not every techSpec contains a softwareDesignSpec. Correct?

Reply:
Right... Changing it

Reply:
I have discovered that the previous code was not using the `getResolvedTechn...` methods so it was getting all of them. Fixed now